### ISSUE TITLE
在特征提取阶段添加一种loss

### ISSUE THREAD

**Issue by apz27** [bubbliiiing/yolov5-pytorch#101]
在不同分辨率的特征图后添加一种loss，想问下需要在代码中哪里添加，怎么添加呢。
loss代码如下：
### input2为特征输入，格式B*F*H*W，scale为该损失权重，设置为1即可

def NewDecorrelationBetweenDim1(input2, scale):
    input2 = input2.view(input2.size(0), input2.size(1), -1)
    input2 = input2.permute(1, 0, 2)
    input_avg = torch.mean(input2, dim=1, keepdim=True)
    # input_avg = torch.mean(input2, dim=[0, 1], keepdim=True)
    input_subAvg = input2 - input_avg
    input_subAvg1 = input_subAvg.reshape(input_subAvg.size(0), -1)
    newTensor = torch.mm(input_subAvg1, input_subAvg1.T) / input_subAvg.size(2)
    input_sum = torch.sum(input_subAvg1.pow(2), dim=1, keepdim=True)
    input_sum = pow(input_sum / input_avg.size(2), 0.5)
    # input_sum = input_sum.view(input_sum.size(0), -1)
    ret_sum = torch.mm(input_sum, input_sum.T).clamp(min=1e-12)
    res = newTensor / ret_sum

    # res_loss = torch.sum(abs(res)) - torch.sum(abs(res.diagonal()))
    res_loss = torch.sum(pow(res, 2)) - torch.sum(pow(res.diagonal(), 2))
    # res_loss = res_loss / ((res.size(0) - 1) * res.size(0))
    res_loss = res_loss / max((res.size(0) - 1) * res.size(0), 1e-12)

    # print(torch.unique(res.diagonal().view(-1).type(torch.), return_counts=True))

    return scale * res_loss
